downstream/assemblies/eda/assembly-installation-eda-controller.adoc

```diff
@@ -4,7 +4,9 @@ ifdef::context[:parent-context: {context}]
 
 = Installation of {EDAcontroller}
 
-The installation of {EDAcontroller} occurs at the same time as the installation for other {PlatformNameShort} components. Like {ControllerName} and {HubName}, the setup includes default settings for specific variables in the inventory files.
+Similar to the {ControllerName} and {HubName} components, the setup for {EDAcontroller} includes default settings for specific variables in the inventory files. However, you must install {ControllerName} before you populate the inventory file with {EDAName} variables. 
```

Comment:
The last sentence isn't true. EDA can be installed at the same time as Controller. Controller doesn't need to be installed ahead of time. If you configure the inventory with both Controller and EDA, the installer will install Controller first that way EDA will have the URL to that Controller environment because the installer knows it exists.

Adding that sentence as is I believe is going to add more confusion.

Reply:
Thanks, @jamesmarshall24 . I can remove the sentence I added and describe things in alignment with your explanation. I'll send it for a second look; just let me know if the new wording is clear.